### PR TITLE
[RISCV] Support `svukte` extension

### PR DESCRIPTION
**Pull request by 4vtomat**
This is the extension for "Address-Independent Latency of User-Mode
Faults to Supervisor Addresses".
Spec: https://github.com/riscv/riscv-isa-manual/pull/1564, https://lf-riscv.atlassian.net/browse/RVS-2977
The spec states that the `svukte` depends on `sv39`, but we don't have
`sv39` yet, so I didn't add it to the implied list.

### CODE REVIEW

**Comment by 4vtomat** [comment]
The spec is frozen, but the PR is still version 0.3, I'll follow the spec to update the version.

**Comment by topperc** [comment]
Missing RISCVUsage.rst

**Comment by 4vtomat** [comment]
> RISCVUsage.rst

Updated!

File: llvm/docs/RISCVUsage.rst [hunk 1]
```diff
@@ -158,6 +158,7 @@ on support follow.
      ``Svinval``       Assembly Support
      ``Svnapot``       Assembly Support
      ``Svpbmt``        Supported
+     ``Svukte``        Supported
```
[the remark this answers is not in the document]
Comment (4vtomat) [reply llvm/docs/RISCVUsage.rst:161]: I thought it's frozen, hasn't it?

[the remark this answers is not in the document]
Comment (topperc) [reply llvm/docs/RISCVUsage.rst:161]: It has to be experimental until ratified. The ratification process can still change a frozen spec.

[the remark this answers is not in the document]
Comment (4vtomat) [reply llvm/docs/RISCVUsage.rst:161]: I see, let me change it~

File: llvm/docs/RISCVUsage.rst [hunk 2]
```diff
@@ -330,6 +329,9 @@ The primary goal of experimental support is to assist in the process of ratifica
 ``experimental-smctr``, ``experimental-ssctr``
   LLVM implements the `1.0-rc3 specification <https://github.com/riscv/riscv-control-transfer-records/releases/tag/v1.0_rc3>`__.
 
+``experimental-svukte``, ``experimental-svukte``
```
Comment (topperc) [inline llvm/docs/RISCVUsage.rst:332]: No need to mention ``experimental-svukte`` twice. The one above that I assume you copied is smctr and ssctr.

Comment (topperc) [reply llvm/docs/RISCVUsage.rst:332]: smctr and ssctr are each mentioned 1 time.

Comment (4vtomat) [reply llvm/docs/RISCVUsage.rst:332]: Oh, I see

**Review by topperc — APPROVED** [review]
LGTM

**Pull request merged**